services/minespace-web/src/components/dashboard/mine/noticeOfDeparture/NoticeOfDeparture.tsx

```diff
@@ -47,14 +48,14 @@ interface NoticeOfDepartureProps {
 }
 
 export const NoticeOfDeparture: FC<NoticeOfDepartureProps> = (props) => {
-  const { mine, nods, permits } = props;
+  const { mine } = useContext<{ mine: IMine }>(SidebarContext);
```

Comment:
I like this.  We'll need to watch that we remember to use it rather than the old method!